backend/taco_backend.py

```diff
@@ -18,6 +18,43 @@
 import sparse
 from .core import Backend
 
+# Hack for @ operator for simplicity. No dimension checks.
+setattr(pt.tensor, '__matmul__', lambda x, y: TacoBackend.dot(x, y))
+
+
+def _scalarfunc(t):
+    if len(t.shape) == 0:
+        # TODO(yejiayu): Extend this to support any type casts.
+        return float(t[0])
+    else:
+        raise TypeError("Only rank 0 can be converted to scalars.")
+
+
+setattr(pt.tensor, '__float__', _scalarfunc)
```

Comment:
could you remind me where is this used?

Reply:
This guy: https://github.com/LinjianMa/AutoHOOT/blob/master/utils.py#L474

Reply:
I see